mbank-cli

```diff
@@ -1225,8 +1225,14 @@ sub do_login  ## no critic (ExcessComplexity)
         $error_code eq ''
             or scraping_error("login.2fa.init.error: $error_code");
         my $auth_mode = $data->{AuthMode} // '';
+        my $operation_date = $data->{OperationDate} // '';
+        $operation_date = parse_timestamp($operation_date)
```

Comment:
```suggestion
        $operation_date = parse_timestamp($operation_date, discard_time => 1)
```
and then you won't need to fiddle with the regexp.
Without `discard_time`, the function checks that the time is all zeros. (Yes, this is all awful.)

Reply:
https://github.com/jwilk/mbank-cli/pull/77/commits/992f258de7e570c5fd6b0b8f7f4fc36f313daa88